frontend/src/selectors/staticContentSelectors.js

```diff
@@ -18,6 +18,12 @@ export const {
   getIncidentFollowupActionOptions,
 } = staticContentReducer;
 
+// removes all expired compliance codes from the array
+export const getCurrentComplianceCodes = createSelector(
+  [getComplianceCodes],
+  (codes) => codes.filter((code) => new Date(code.expiry_date) > new Date())
+);
```

Comment:
We should chat with someone from business to find out if we should be excluding expired compliance codes at the API level

Reply:
Yes, agreed. Chatted with Aaron and he was happy to remove the expired, but has no preference if its client or server side.

Reply:
Okay. It's really just a matter of whether or not the expired (or future) values should be "exposed" on principle. Would we want to provide that data to an external consumer of the API? If not, we should move it to the server. This would be my preference, considering how we handle `active_ind` currently (filtering by it before returning data from the API). Mind you, I was told that the codes were being treated differently, so I'm not sure if we need to confirm that with someone.

It's probably OOS for this PR. Not a big job to add the logic to the backend and then just remove this filter in the future.